hawc/apps/assessment/api/viewsets.py

```diff
@@ -253,11 +253,18 @@ def get_queryset(self):
         return self.model.objects.all()
 
 
-class Assessment(AssessmentViewSet):
+class Assessment(AssessmentViewSet, AssessmentEditViewSet):
```

Comment:
remove AssessmentViewSet inheritance?

Reply:
isn't it used for the other actions (public, endpoints, logs)?

Reply:
No, I dont think so, the AssessmentEditViewSet has all the same configuration. https://github.com/shapiromatron/hawc/blob/eb4aafd3c8eeb1a35e609479f1a20420266d5d93/hawc/apps/assessment/api/viewsets.py#L182 

The inheritance/naming pattern with these is confusing for sure.